src/index.tsx

```diff
@@ -22,13 +21,13 @@ const applicationRoot = document.getElementById('root') as HTMLElement;
 const redirectToError = () => window.location.assign('./error.html');
 
 /*
-// Add Accessibility reporting in development via Chrome console through React-axe
+// Add Accessibility reportin   g in development via Chrome console through React-axe
```

Comment:
looks like this file has some unnecessary changes, could just remove it from the PR